test/data/test_utils.py

```diff
@@ -45,3 +46,11 @@ def test_get_tokenizer_toktokt(self):
             data.get_tokenizer(1)
         with self.assertRaises(ValueError):
             data.get_tokenizer("some other string")
+
+    def test_text_nomalize_function(self):
```

Comment:
You should make this more rigorous by implementing the bash function and running it across all of AG_NEWS. I think it's worth doing that, because it's easy to miss something. In the future we might update this test as the basic normalization function becomes more sophisticated.